invenio_records_resources/records/systemfields/files/field.py

```diff
@@ -23,42 +23,32 @@
             'figure.png',
             'data.zip',
         ],
-        'meta': {
```

Comment:
I've removed the ``meta`` attribute in favour of integrating the metadata into the file itself. This gives also a cleaner way to dump/load a file record.